man/systemd.network.xml

```diff
@@ -210,6 +212,13 @@
           <para>Identity Association Identifier for the interface, a 32-bit unsigned integer.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+      <listitem>
+        <para>Note that if any interface where no static IPv6 address, DHCPv6 or no IPV6LL is on shall
```

Comment:
… where no static IPv6 address is configured, and neither DHCPv6 nor IPv6LL is enabled, shall be considered …